corehq/apps/translations/app_translations.py

```diff
@@ -862,7 +862,7 @@ def has_translation(row_, langs):
 def escape_output_value(value):
     try:
         return etree.fromstring("<value>{}</value>".format(
-            re.sub(r"(?<!/)>", "&gt;", re.sub(r"<(\s*)(?!output)", "&lt;\\1", value))
+            re.sub("(?<!/)>", "&gt;", re.sub("<(\s*)(?!output)", "&lt;\\1", value))
```

Comment:
why this change?

Reply:
Not intentional, I must have done a bad rebase incorporating https://github.com/dimagi/commcare-hq/pull/23403

Fixing in 7150aa0e9fc

Reply:
you missed the second "r" in the fix.